### PR TITLE
"__compare" endpoint to compare between versions

### PR DESCRIPTION
**Pull request by mariabg**

### CODE REVIEW

File: data/comparison-images.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+[ "/images/raw/http%3A%2F%2Fim.ft-static.com%2Fcontent%2Fimages%2Fa60ae24b-b87f-439c-bf1b-6e54946b4cf2.img?source=test",
```
Comment (rowanmanning) [inline data/comparison-images.json]: Could we tab-indent this file?

File: lib/routes/__compare.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+// Generate image variants
+const imagesSrc = require('../../data/comparison-images.json');
+
+// imagesSrc.forEach((img) => {
+// 	return img;
+// });
```
Comment (rowanmanning) [inline lib/routes/__compare.js]: Commented code should be removed :)

File: lib/routes/__compare.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+// original stuff
+module.exports = app => {
+	app.get('/__compare', (request, response) => {
```
Comment (rowanmanning) [inline lib/routes/__compare.js]: I don't think we need the double underscores here. I'd be happy if it was just `/compare`, as it'd be considered a public URL and we'll probably link to it

**Comment by rowanmanning** [comment]
I made a couple of comments but this looks great. Really useful to have 👍

**Comment by rowanmanning** [comment]
Actually running locally it's odd, the table cells aren't the same widths and it means that the images are scaling differently. It might be good to explicitly set the table cells to be 50% wide?

**Comment by mariabg** [comment]
I made some changes, please checkout if they are what you had in mind :)

**Comment by rowanmanning** [comment]
Looks great 👍 thanks!

**Comment by mariabg** [comment]
Merge?

**Pull request merged**